notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>nit - "looks odd than" isn't quite the right phrasing here - because it's relative you'd say "odder" but honestly, I don't love that here. What about it is odd? perhaps you can use a different word to describe why it's odd compared to the taxi data?</p><p> </p><p>nit - second paragraph - trips spending "too much time than trip_distance" also unfortunately doesn't work - do you instead mean something like "the trip time was longer than expected given the trip distance"? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>